core/trino-main/src/main/java/io/trino/memory/MemoryManagerConfig.java

```diff
@@ -35,13 +35,13 @@
         "resources.reserved-system-memory"})
 public class MemoryManagerConfig
 {
-    public static final String FAULT_TOLERANT_TASK_MEMORY_CONFIG = "fault-tolerant-task-memory";
+    public static final String FAULT_TOLERANT_TASK_MEMORY_CONFIG = "fault-tolerant-execution-task-memory";
 
     // enforced against user memory allocations
     private DataSize maxQueryMemory = DataSize.of(20, GIGABYTE);
     // enforced against user + system memory allocations (default is maxQueryMemory * 2)
     private DataSize maxQueryTotalMemory;
-    private DataSize faultTolerantTaskMemory = DataSize.of(1, GIGABYTE);
+    private DataSize faultTolerantExecutionTaskMemory = DataSize.of(4, GIGABYTE);
```

Comment:
Ideally it would be to define it as a fraction of worker heap but we do not have this information on coordinator statically available. Also every worker can be of different size theoretically.